src/middlewared/middlewared/plugins/system.py

```diff
@@ -376,23 +376,27 @@ async def product_type(self):
         """
         Returns the type of the product.
 
-        CORE - TrueNAS Core, community version
-        ENTERPRISE - TrueNAS Enterprise, appliance version
-        SCALE - TrueNAS SCALE
+        CORE - TrueNAS Core, community version (freeBSD)
```

Comment:
Can we remove the freebsd and linux mentions since this is public and API readers shouldnt care about the OS?